package.json

```diff
@@ -1,31 +1,22 @@
 {
   "name": "posthtml-include",
   "version": "1.4.0",
-  "description": "Include file in HTML",
-  "main": "index.js",
-  "dependencies": {
-    "posthtml": "^0.12.0",
-    "posthtml-expressions": "^1.1.1",
-    "posthtml-parser": "^0.4.2"
-  },
-  "devDependencies": {
-    "chai": "^4.2.0",
-    "istanbul": "^0.4.5",
-    "jscs": "^3.0.7",
-    "jshint": "^2.11.0",
-    "mocha": "^7.1.0"
+  "description": "Include files in HTML",
+  "license": "MIT",
+  "author": "Ivan Voischev <voischev.ivan@ya.ru>",
+  "bugs": "https://github.com/posthtml/posthtml-include/issues",
+  "homepage": "https://github.com/posthtml/posthtml-include",
+  "repository": "posthtml/posthtml-include",
+  "main": "lib",
+  "engines": {
+    "node": ">=10"
```

Comment:
may be we should add `files : ['lib']` prop cause `.npmignore` file has been removed

thoughts ?

Reply:
Added 👍